common/app/views/emailEmbed.scala.html

```diff
@@ -35,4 +35,8 @@
     <body>
         @body
     </body>
+    <script src="https://interactive.guim.co.uk/libs/iframe-messenger/iframeMessenger.js"></script>
```

Comment:
I'm not sure how I feel about having a functionality of the newsletter iframe relying on a script from the interactive team.

On one hand we avoid code duplication, but on the other we open the door to another team breaking the behaviour of our embed without their knowledge.

Have you chatted about this with anyone else, or is there something I'm missing?

Reply:
This came from a discussion with Dotcom and it looks like this is at least what is being done for atoms etc., so it seems reasonable to rely on it atm. I agree that it feels a bit risky but I feel like this is the closest we'll get to a standardised way of handling iframe resizing.

Reply:
As long as it's been discussed with dotcom it's all good!